lib/nimble_options.ex

```diff
@@ -93,6 +93,8 @@ defmodule NimbleOptions do
 
     * `{:fun, arity}` - Any function with the specified arity.
 
+    * `{:one_of, choices}` - A value that is a member of one of the `choices`.
```

Comment:
Reading the docs... now I am thinking `:choices` would be the better name? :D

Reply:
@josevalim I think `{:choices, ...}` looks weird when used in practice. `{:one_of, [:flush, :bulk]}` reads better to me than `{:choices, [:flush, :bulk]}`. I don't have a strong preference though, so just tell me what you prefer and I'll implement that :)

Reply:
agreed about :one_of. another suggestion would be `{:in, [:flush, :bulk]}`? but not strong opinion either.